nncf/openvino/statistics/collectors.py

```diff
@@ -195,6 +196,11 @@ def get_output_names(self, target_node_name: str, port_id: int) -> List[str]:
         return [get_result_node_name(target_node_name, port_id)]
 
 
+class OVRawReducer(RawReducer):
+    def get_output_names(self, target_node_name: str, port_id: int) -> List[str]:
+        return [get_result_node_name(target_node_name, port_id)]
```

Comment:
`get_output_names` method does not exist anymore, there is no need to make a backend-specific realization for this reducer

Reply:
Then, why wasn't it dropped for other reducers? Why do we have backend reducers now?

Reply:
I don't get what should I do with this. Do I need to remove `OVRawReducer` and `OVNoopReducer` as well?

Reply:
Yes, if you don't mind. I made a PR recently https://github.com/openvinotoolkit/nncf/pull/2391 and forgot to delete OVNoopReducer, PTNoopReducer and ONNXNoopReducer

Reply:
`get_output_names` also exists for other Reducers and needs for `TensorCollector.get_output_info`.
Why do you suggest to remove it?

Reply:
Please rebase your PR

Reply:
Ok, then I've got it.
What should we do with the [ONNX backend](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/onnx/statistics/collectors.py#L199-L232) and [Torch backend](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/tensor_statistics/collectors.py#L226-L259) then? Why should we keep all these classes?

Reply:
1) OV has inplace reducers implementation and other backend does not have it
2) method `_get_processor` is still backend-specific

Reply:
For some reducers, we're going to have common implementations, for some - not. That confuses me. And there is still no answer as to why we can't drop **pass** reducers for ONNX and Torch.

Reply:
> For some reducers, we're going to have common implementations, for some - not. That confuses me. 

Reducers which do nothing except filtering can be backend-agnostic, reducers which do something with input tensor cannot be backend-agnostic yet. 

We can drop PTNoopReducer, OVNoopReducer, ONNXNoopReducer and OVRawReducer, PTRawReducer, ONNXRawReducer, no problems.  I suggest to drop them

Reply:
We can also drop ONNXMinReducer, ONNXMaxReducer, ONNXAbsMaxReducer, etc. since they do nothing, can't we?

Reply:
Where is the implementation [here](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/onnx/statistics/collectors.py#L199-L232)?

Reply:
Please see this https://github.com/openvinotoolkit/nncf/blob/develop/nncf/onnx/statistics/collectors.py#L191-L197

Reply:
What a mess with these classes.

Reply:
Awful, I agree